libraries/AP_Logger/LogStructure.h

```diff
@@ -1327,7 +1329,7 @@ LOG_STRUCTURE_FROM_AVOIDANCE \
     { LOG_SIMSTATE_MSG, sizeof(log_AHRS), \
       "SIM","QccCfLLffff","TimeUS,Roll,Pitch,Yaw,Alt,Lat,Lng,Q1,Q2,Q3,Q4", "sddhmDU????", "FBBB0GG????", true }, \
     { LOG_TERRAIN_MSG, sizeof(log_TERRAIN), \
-      "TERR","QBLLHffHH","TimeUS,Status,Lat,Lng,Spacing,TerrH,CHeight,Pending,Loaded", "s-DU-mm--", "F-GG-00--", true }, \
+      "TERR","QBLLHffHHf","TimeUS,Status,Lat,Lng,Spacing,TerrH,CHeight,Pending,Loaded,HCorr", "s-DU-mm--m", "F-GG-00--0", true }, \
```

Comment:
.. and here too let's maybe call this column "RefOfs".  I don't mind if we call it a correction or an offset but let's pick one so people can more quickly realise they are the same thing.

Reply:
done